src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -687,63 +683,68 @@ private static boolean getPreviousState(String eachState) {
     @SecurityRequirement(name = "Bearer Authentication")
     public AboutToStartOrSubmitCallbackResponse copyManageDocsForTabs(
         @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestHeader(PrlAppsConstants.SERVICE_AUTHORIZATION_HEADER) String s2sToken,
         @RequestBody CallbackRequest callbackRequest
     ) {
-        CaseData caseData = CaseUtils.getCaseData(callbackRequest.getCaseDetails(), objectMapper);
```

Comment:
We can remove this method completely. it was used by older manage documents event which was replaced with new event back in July 2023